drivers/docker/driver.go

```diff
@@ -745,10 +745,13 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		hostConfig.CPUQuota = int64(task.Resources.LinuxResources.PercentTicks*float64(driverConfig.CPUCFSPeriod)) * int64(numCores)
 	}
 
+	// disable swapiness
+	var swapiness int64 = 0
+	hostConfig.MemorySwappiness = &swapiness
```

Comment:
How does this interact with the non-windows MemorySwap path?

Reply:
It probably safer to be more conservative here.
https://github.com/hashicorp/nomad/pull/6522/files#diff-2b17483fc1838ce536b7f8e6f6f6ea8e

Reply:
It's back to what I was before.